oaknational/Oak-Web-Application
# [CUR-642] Updated `jose` dependency

## Description

- Updated `jose` as per https://github.com/oaknational/Oak-Web-Application/security/dependabot/91

## Issue(s)

`CUR-642`

## How to test
Check app can make/generate graphql, also mux works correctly. As `jose` is deps of `@mux/mux-node` and `@mux/mux-node`
